utils/zerovec/Cargo.toml

```diff
@@ -37,6 +38,7 @@ serde = { version = "1.0", default-features = false, features = ["alloc"], optio
 # and all 0.7 versions, but not further.
 yoke = { version = ">=0.6.0, <0.8.0", path = "../yoke", optional = true }
 twox-hash = { version = "1.6", default-features = false, optional = true }
+schemars = "0.8.16"
```

Comment:
issue: this must be an optional dependency

Reply:
Yes, and please add it to the top-level Cargo.toml (ideally wait until https://github.com/unicode-org/icu4x/pull/4844 is merged)